products/bash/release-ace-integration-server.sh

```diff
@@ -13,36 +13,59 @@
 #   -n : <namespace> (string), Defaults to "cp4i"
 #   -r : <is_release_name> (string), Defaults to "ace-is"
 #   -i : <is_image_name> (string), Defaults to "image-registry.openshift-image-registry.svc:5000/cp4i/ace-11.0.0.9-r2:new-1"
+#   -z : <tracing_namespace> (string), Defaults to ""
```

Comment:
Need to doc the -t option, and what "" means for -z (I.e. same namespace as -n)

Reply:
done